utils/python/CIME/batch_maker.py

```diff
@@ -108,33 +109,32 @@ def _set_task_info(self):
     def _set_queue(self):
         self.queue = self.env_batch.get_value("JOB_QUEUE", subgroup=self.job)
         self.ccsm_estcost = self.case.get_value("CCSM_ESTCOST")
-
         self.wall_time_max = None
         if self.queue:
             return
 
         # Make sure to check default queue first.
-        self.queue = self.config_machines_parser.get_default_queue()
```

Comment:
I found this code really confusing - sometimes self.queue was a string and other times it was a node.   Now it's always a string.